server/scsynth/SC_ComPort.cpp

```diff
@@ -328,18 +328,18 @@ class SC_TcpConnection:
 	{
 		namespace ba = boost::asio;
 		async_read(socket, ba::buffer(&OSCMsgLength, sizeof(OSCMsgLength)),
-				   boost::bind(&SC_TcpConnection::handleLengthReceived, shared_from_this(),
-							   ba::placeholders::error,
-							   ba::placeholders::bytes_transferred)
-				   );
+		           boost::bind(&SC_TcpConnection::handleLengthReceived, shared_from_this(),
+		           ba::placeholders::error,
+		           ba::placeholders::bytes_transferred)
+		          );
```

Comment:
I think you messed up the alignment on this one. Lines 332 and 333 should be aligned with the opening paren of line 331. Maybe the inner close paren should be moved to its own line as well.